### ISSUE TITLE
Simple Spaces to Dict

### ISSUE THREAD

**Issue by rusu24edward** [rusu24edward/amber#75]
Create a wrapper that converts spaces to Dict keyed off the agents ids.

MultiBinary and MultiDiscrete: Ensure the length is same as number of agents and then each element in array is just the value in the dictionary for that agent-key.
Discrete: This is one-hot MultiBinary, so just follow steps above.

Custom mappings:
Every agent can have a custom mapping defined that maps agent ids to the location in the array. So the length of the array does not need to equal the number of agents, just the number of agents in the custom mapping.

Comes out of #74.

**Comment by rusu24edward** [comment]
Begs a bigger question of the usefulness of wrappers. Instead of wrapping the environment, each environment can use transformations to turn the data into something useful. So instead of a simple to dict wrapper (which would need to then be updated to handle dict{simple} to dict{dict}), why not just have the environment wrap the action when it comes in and work forward, like so:
```
def step(self, action_dict):
    action_dict = some_wrapper(action_dict, params)
    ... # The rest of the step function using the transformed action

**Comment by rusu24edward** [comment]
Resolved in #79 with a different approach.
